packages/nft-base-api/src/controllers/assets.ts

```diff
@@ -61,6 +62,26 @@ export class AssetsController extends BaseController {
 		return this.respondWithBlockResource(transaction, request.query.transform, AssetResource);
 	}
 
+	public async showWalletAssets(request: Hapi.Request, h: Hapi.ResponseToolkit) {
```

Comment:
Return `Promise<any>` so we don't violate `@typescript-eslint/explicit-module-boundary-types` rule